applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -177,9 +177,6 @@ class TrilinosBlockBuilderAndSolver
 
         KRATOS_ERROR_IF(!pScheme) << "No scheme provided!" << std::endl;
 
-        // Resetting to zero the vector of reactions
-        TSparseSpace::SetToZero(*BaseType::mpReactionsVector);
```

Comment:
this is of size 0 in the block builder and solver right?

Reply:
i mean this is used by the elimination builder and solver but not by the block builder and solver.